models/check.ts

```diff
@@ -125,17 +126,19 @@ export class Check {
 
   /**
    * Must either be an address ID or an inline object with correct address parameters.
-   * @type {string | AddressEditable}
+   * @type {any}
```

Comment:
@mwj8410  This should be string right?

Reply:
correct. Captured as [DXP-789](https://lobsters.atlassian.net/browse/DXP-789)